cvat/apps/engine/signals.py

```diff
@@ -3,21 +3,17 @@
 # SPDX-License-Identifier: MIT
 import shutil
 
+from django.contrib.auth.models import User
 from django.db.models.signals import post_delete, post_save
 from django.dispatch import receiver
-from django.contrib.auth.models import User
 
-from .models import (
-    Data,
-    Job,
-    StatusChoice,
-    Task,
-    Profile,
-)
+from .models import (CloudStorage, Data, Job, Profile, Project,
+    StatusChoice, Task)
 
 
-@receiver(post_save, sender=Job, dispatch_uid="update_task_status")
-def update_task_status(instance, **kwargs):
+@receiver(post_save, sender=Job,
+    dispatch_uid=__name__ + ".update_task_status_handler")
```

Comment:
Where do these names come from? I suggest to declare they somewhere or provide a function to access them.

Reply:
Directories are always different. They have ID of objects from DB inside. It is not possible.

Reply:
Okay. As I understand, this id is supposed to be unique and should not be used from anywhere. Maybe it would be better to supply an UUID then?

Reply:
I'm not sure it will be better than the proposed solution.